client/dashboard/profile-wizard/style.scss

```diff
@@ -0,0 +1,27 @@
+/** @format */
+
+.woocommerce-profile-wizard__body {
+	background: #1d232f;
+
+	.woocommerce-profile-wizard__header-title {
+		color: #fff;
+	}
+
+	#woocommerce-layout__primary {
+		text-align: center;
+		margin-left: auto;
+		margin-right: auto;
+		width: 70%;
```

Comment:
Should we use a pixel based max-width here or min-width media query?  Might be a touch tight on smaller screens once we start adding more content.

```suggestion
		@include breakpoint( '>960px' ) {
			width: 70%;
		}
```
<img width="340" alt="Screen Shot 2019-03-28 at 12 23 28 PM" src="https://user-images.githubusercontent.com/10561050/55130099-93c86f80-5154-11e9-9a21-d4aec1afa68d.png">